tests/projects/utils.py

```diff
@@ -71,9 +71,8 @@ def docker_example_base_image():
                 path=cwd,
             )
         except BuildError as build_error:
-            for chunk in build_error.build_log:
-                print(chunk)
+            for _ in build_error.build_log:
+                pass
```

Comment:
```suggestion
            for chunk in build_error.build_log:
                chunk  # pylint: disable=print-function
```

One last thing. Can we keep this print which I think is useful for debugging?

Reply:
Keep print or use logger? :thinking:

Reply:
Let's use print.

Reply:
Info level works for this?

Reply:
Done with info log